src/Scripting/VM/VMException.hpp

```
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <format>
+#include <string>
+#include <exception>
+
+#define VM_ASSERT_CONFIG(cond, excMsg, ...) \
```

Clang (14.0.6)

[warning] variadic macro 'VM_ASSERT_CONFIG' used; consider using a 'constexpr' variadic template function

(cppcoreguidelines-macro-usage)